app/client/Table.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react'
+import './Table.css'
+
+class Table extends Component {
+  constructor(props) {
+    super(props) //since we are extending class Table so we have to use super in order to override Component class constructor
+    this.hours = Object.keys(this.props.data)
+    this.days = ["Mon", "Tue", "Wed", "Thu", "Fri"]
+  }
+
+  renderTableRow() {
+    let rows = this.hours.map((hour) => [...[hour], ...Object.values(this.props.data[hour])])
+    return rows.map((row, i) => <tr key={i}>{row.map((el, idx) => <td key={idx}>{el}</td>)}</tr>)
```

Comment:
Try to alocate this in a variable before u return, them you can tell to the others dev's what is it easily what this suppose to be/do using a good name for the var!